tests/karma.config.js

```diff
@@ -78,10 +93,10 @@ module.exports = function(config) {
 
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['Chrome_Puppeteer'],
+    browsers: ['ChromeHeadless_Custom'],
 
     customLaunchers: {
-      Chrome_Puppeteer: {
+      ChromeHeadless_Custom: {
```

Comment:
This is just a name change to align this file with the corresponding one in the client repo.